### PR TITLE
onboarding chapter 3 content for sdks and apis

### PR DESCRIPTION
**Pull request by charleszipp**
Adds learning content for the digital twins sdks and apis

### CODE REVIEW

File: docs/onboarding/03-sdks-and-apis.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Chapter 3: Digital Twin SDK's and API's
+
+Azure digital twins provides a C# SDK for programmatic management of models and twins. The following covers the capabilities of the SDK's and APIs
```
Comment (jackiebecker) [inline docs/onboarding/03-sdks-and-apis.md]: ADT also offers a Java SDK. Perhaps worth mentioning, although you can always say "these onboarding documents all focus around features in C#, so bear that in mind if you are developing a Java project" if you want to make it clear we're not sure about feature parity

File: docs/onboarding/03-sdks-and-apis.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- The SDK is backed by Azure Digital Twins API. The API could be leveraged when a client SDK is not available.
+- The SDK authenticates using the Azure.Identity library (DefaultAzureCredentials preferred).
+- Twins can be represented in type-safe code as POCO's and JSON Serialization.
+
+## Learning Exercises
+
+Using the models created from [Learning Exercises in Chapter 2](02-digital-twin-definition-language.md), create a console app that is able to do the following operations.
+
+1. Creates a model
+1. Checks if a model exists before attempting to create it.
```
Comment (jackiebecker) [inline docs/onboarding/03-sdks-and-apis.md]: Should 1 & 2 be switched, just logically?

File: docs/onboarding/03-sdks-and-apis.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+1. Updates a property of the twin created in #3
+1. Reads a property from the twin created in #3 and writes the value to the console.
+
+## Experiments
+
+- Write a unit test that validates a model is only created if it doesn't already exist.
+
+## Things to Consider
+
+- How would need to be added, if anything, to facilitate unit testing?
```
Comment (jackiebecker) [inline docs/onboarding/03-sdks-and-apis.md]: Nit: "what" would

**Review by jackiebecker — APPROVED** [review]

File: docs/onboarding/03-sdks-and-apis.md (hunk 3 above)
Comment (charleszipp) [reply docs/onboarding/03-sdks-and-apis.md]: fixed. good catch!

File: docs/onboarding/03-sdks-and-apis.md (hunk 2 above)
Comment (charleszipp) [reply docs/onboarding/03-sdks-and-apis.md]: just consolidated these. after re-reading seems a bit too granular.

File: docs/onboarding/03-sdks-and-apis.md (hunk 1 above)
Comment (charleszipp) [reply docs/onboarding/03-sdks-and-apis.md]: Clarified this doc and added your suggestion to the root onboarding readme.

**Pull request merged**